.github/workflows/build.yml

```diff
@@ -98,6 +98,15 @@ jobs:
           token:  ${{ secrets.ACTIONS_API_ACCESS_TOKEN }}
           inputs: '{"review": "${{ github.event.pull_request.number }}", "sha": "${{ env.IMAGE_TAG }}"}'
 
+      - name: Run Snyk to check Docker image for vulnerabilities
+        uses: snyk/actions/docker@master
+        continue-on-error: true
```

Comment:
Does this mean the build doesn't fail if vulnerabilities are found or that other types of error are ignored?

Reply:
Currently it wouldn't fail the build if the step finds vulnerabilities or fails for another reason.
Also apply is currently clear for vulnerabilities in the base image.